protocols/kad/src/behaviour.rs

```diff
@@ -464,7 +462,7 @@ where
             mode: Mode::Client,
             auto_mode: true,
             no_events_waker: None,
-            bootstrap_status: BootstrapStatus::new(config.bootstrap_interval),
+            bootstrap_status: Status::new(config.bootstrap_interval),
```

Comment:
Since there is already a `use crate::{bootstrap, K_VALUE};`

```suggestion
            bootstrap_status: bootstrap::Status::new(config.bootstrap_interval),
```

Reply:
Nice catch.